Composer/packages/lib/shared/src/viewUtils.ts

```diff
@@ -183,19 +183,26 @@ export function getDialogGroupByType(type) {
 
 const truncateSDKType = ($kind) => (typeof $kind === 'string' ? $kind.replace('Microsoft.', '') : '');
 
+const getSchemaTitle = (sdkschema: JSONSchema7, data) => {
+  const rawTitleValue = sdkschema.title as string | ((d) => string) | undefined;
+  // Handle the case that FormUIOption label function overrides sdk.schema.
+  return typeof rawTitleValue === 'function' ? rawTitleValue(data) : rawTitleValue;
```

Comment:
Why would the sdk schema contain a function? Where does this get overwritten?

Reply:
It do bring confusion, I'll make it clear

Reply:
To make the Action title be consistent with Form's title, the FormConfig is merged into sdkSchema object in client/VisualEdtitor.tsx, the `label` option overrides the `title` option.

It can but I didn't make it consume FormConfig directly because I think it's improper to couple with Form uischema.